### PR TITLE
[Sensu] - fixing resources

### PR DESCRIPTION
**Pull request by sstarcher**
I'm not sure how this got missed the first round, but the resources block was entirely missing.

### CODE REVIEW

File: stable/sensu/templates/deployment.yaml [hunk 1]
```diff
@@ -47,6 +47,8 @@ spec:
         args:
           - api        
 {{ toYaml .Values.api.resources | indent 8 }}
```
Comment (lachie83) [inline stable/sensu/templates/deployment.yaml]: It looks like it is there but doesn't have the key. It also looks like the server resources in broken too. Can you fix that as well and yank this line?

**Comment by lachie83** [comment]
Looks to me like the templating for both server and api resources are broken. Need to test to see what's going on here

**Comment by sstarcher** [comment]
@lachie83 that was my mistake I did not stage some of my changes.  It should now have the changes for both resource objects.

**Comment by lachie83** [comment]
@sstarcher no worries at all. That looks much better. I will test and let you know

**Comment by prydonius** [comment]
Thanks @sstarcher, tested and looks ok to me

**Pull request merged**